src/app/Shared/MatchExpressionEvaluator.tsx

```diff
@@ -125,13 +120,28 @@ export const MatchExpressionEvaluator: React.FunctionComponent<MatchExpressionEv
       </StackItem>
       <StackItem>
         { statusLabel }
+        {
+          !props.inlineHint ?
+            <Tooltip content={<div>Hint: try an expression like <br />{ exampleExpression }</div>}>
+              <HelpIcon noVerticalAlign />
```

Comment:
Maybe some space between the help icon and the status label?

![Screenshot from 2022-07-13 11-50-53](https://user-images.githubusercontent.com/68053619/178778601-c48f1cc7-702d-44f3-89b6-2b58fc70ee06.png)

Reply:
![image](https://user-images.githubusercontent.com/3787464/178789409-554d7d4e-8a32-4249-9757-704278d39ef5.png)

Reply:
Looks good!